src/main/io/ledstrip.c

```diff
@@ -1014,7 +1014,7 @@ typedef enum {
 } timId_e;
 
 static timeUs_t timerVal[timTimerCount];
-static bool requiredTimerLayer[timTimerCount];
+static uint16_t disabledTimerMask;
 
```

Comment:
STATIC_ASSERT(timTimerCount <= sizeof(disabledTimerMask) * 8, disabledTimerMask_too_small);

Reply:
I don't understand this.  `timTimerCount` is the last value in the enumeration of timer types and currently equates to max of 10 (depending on USE_ includes).  `disabledTimerMask` is a uint16_t.

Are you saying to add this as a test in case too many timers get added in the future?

Reply:
`timTimerCount` is number of bits needed for timer disabling, `sizeof(disabledTimerMask) * 8` is number of bits available. Ti will fail if `uint16_t` is too small ...

(it's quite easy to overlook mask size when adding enum values)

Reply:
Ok, makes sense.  Good check to prevent mistakes in the future - I'll add it.